pallets/runtime/src/asset.rs

```diff
@@ -703,10 +723,11 @@ decl_module! {
         }
 
         /// Function used to create the checkpoint
+        /// NB can only be called by the owner of the security token i.e owner DID
```

Comment:
This is going to be formatted without a line break in the HTML. Needs a comma after each sentence.